schema/codelists/documentType.csv

```diff
@@ -23,20 +23,20 @@ implementation,physicalProgressReport,Physical progress reports,"Documentation o
 implementation,financialProgressReport,Financial progress reports,"Documentation providing dates and amounts of stage payments made (against total amount) and the source of those payments, including cost overruns, if any. Structured versions of this information can be provided through contract implementation transactions.",,
 implementation,finalAudit,Final audit,Documentation of a final audit carried out at the end of contract implementation.,,
 tender,hearingNotice,Public hearing notice,Documentation of any public hearings that took place as part of the planning for this contracting process.,,
-planning,marketStudies,Market studies,Documentation of any market studies that took place as part of the planning for this contracting process.,,
+planning,marketStudies,Market studies,Documentation of any market studies that took place as part of the planning for the future contracting process.,,
 tender,clarifications,Clarifications to bidders questions,Documentation that provides replies to issues raised in pre-bid conferences or an enquiry processes.,,
 tender,shortlistedFirms,Shortlisted firms,Documentation providing information on shortlisted firms. Structured versions of this information can be provided using the bids extension.,,
-"planning, implementation",environmentalImpact,Environmental impact,"Documentation of assessments of the environmental impacts (e.g. impacts on flora, fauna & woodlands, areas of natural beauty, carbon emissions etc.) and mitigation measures (e.g. pollution control, low carbon solutions, sustainable timber etc.) for the contracting process.",,
+"planning, implementation",environmentalImpact,Environmental impact,"Documentation of assessments of the environmental impacts (e.g. impacts on flora, fauna & woodlands, areas of natural beauty, carbon emissions etc.) and mitigation measures (e.g. pollution control, low carbon solutions, sustainable timber etc.) for the future contracting process.",,
```

Comment:
I would try to avoid using activities, see above. However, in this issue, I think we have an easy way out: somewhat similarly to the discussion on budgets, the environmental impact isn't of the of the process per se, but rather of the actual purchase (i.e. items). So I would replace the "for contracting process" with "of the works, goods or services".

Proposed in https://github.com/open-contracting/standard/pull/1513/commits/be8c64f5a9bb0998c0e4d93000023f74efec0fe6. Related issue opened in https://github.com/open-contracting/standard/issues/1521#issue-1242889108.